u19_pipeline/recording.py

```diff
@@ -0,0 +1,181 @@
+import datajoint as dj
+from u19_pipeline import lab, task, subject, acquisition
+
+schema = dj.schema('u19_recording')
+
+
+
+@schema
+class RecordingModality(dj.Lookup):
+     definition = """
+     recording_modality:        varchar(64)          # modalities for recording (ephys, imaging, video_recording, etc.) 
+     ---
+     modality_description:      varchar(255)         # description for the modality
+     root_direcory:             varchar(255)         # root directory where that modality is stored (e.g. ephys = /braininit/Data/eletrophysiology)
+     recording_file_extensions: blob                 # file extensions specific for this modality
+     """
+     contents = [
+        ['ephys',             '', '/braininit/Data/electrophysiology', ['ap.bin', 'ap.meta']],
+        ['imaging',           '', '/braininit/Data/imaging', ['.avi', '.tiff','.tif']],
+        ['video_acquisition', '', '/braininit/Data/video_acquisition', ['.avi', '.mp4']]
+     ]
+
+
+@schema
+class PreprocessingParamSet(dj.Lookup):
+    definition = """
+    # Parameter set to be used in the preprocessing steps
+    preprocess_paramset_idx:  smallint
+    ---
+    -> RecordingModality    
+    preprocess_paramset_desc: varchar(128)
+    preprocess_param_set_hash: uuid
+    unique index (preprocess_param_set_hash)
+    preprocessing_params: longblob  # dictionary of all applicable parameters
```

Comment:
```suggestion
    preprocess_paramset_hash: uuid
    unique index (preprocess_paramset_hash)
    preprocessing_paramset: longblob  # dictionary of all applicable parameters
```

For consistency, I am referring to everything as `paramset`.

Reply:
Hi @Alvalunasan, if you agree, let's also change the attributes in the `ProcessingParamSet` table.  Thanks.

Reply:
Will do